src/js/shared/utils.js

```diff
@@ -31,3 +31,21 @@ async function setCustomFonts() {
     document.fonts.add(font);      
   }
 }
+
+function preventDefaultBehavior(clickEvt) {
```

Comment:
Heh, I'm sure I've seen this and the function below before... Possibly worth seeing if it can be removed elsewhere (or alternatively, here), since shadowing can lead to very confusing behaviour when debugging something. WHY ISN'T THIS CONSOLE.LOG SHOWING UP DAMMIT! :P

Reply:
Heh. I made this the only instance in 4344dc8e29ad93376f4af2c58eebb7a81182ed08